frontend/src/components/videoPlayer/AddComment.js

```diff
@@ -20,27 +24,99 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const AddComment = () => {
+const AddComment = ({ data }) => {
   const classes = useStyles();
+  const [comment, setComment] = useState('');
+  const [alert, setAlert] = useState({
+    show: false,
+    message: '',
+    severity: '',
+  });
+
+  const onChangeComment = (e) => {
+    const comment = e.target.value;
+    setComment(comment);
+  };
+
+  const handleComment = async (e) => {
+    e.preventDefault();
```

Comment:
add additional check here that if comment has wrong format (too long or emty), you can show the error immediately without sending it to the backend.